### PR TITLE
 SSB: Implement Buffs, Nerfs, and Bugfixes

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

**Comment by ImLycan** [comment]
hey, can you update ranks too?
False should be &
Lycanium Z should be +

File: mods/ssb/moves.js [hunk 1]
```diff
@@ -1298,18 +1302,17 @@ let BattleMovedex = {
 	// False
 	frck: {
 		accuracy: true,
-		basePower: 1000,
+		basePower: 0,
```
Comment (HoeenCoder) [inline mods/ssb/moves.js]:
```suggestion
		basePower: 1000,
```

Comment (HoeenCoder) [reply mods/ssb/moves.js]: Is more fun to leave as is (for /dt ect)

Comment (KrisXV) [reply mods/ssb/moves.js]: Ugh I keep hitting resolve 

Then this throws an error if it koes the user first and I don’t think there needs to be s target check

Comment (HoeenCoder) [reply mods/ssb/moves.js]: Alright, leave as 0 then

**Pull request merged**